targets/TARGET_ONSEMI/TARGET_NCS36510/gpio_api.c

```diff
@@ -143,16 +140,17 @@ void gpio_mode(gpio_t *obj, PinMode mode)
 void gpio_dir(gpio_t *obj, PinDirection direction)
 {
     /* Enable the GPIO clock */
-    CLOCK_ENABLE(CLOCK_GPIO);
+    if(!CLOCK_IS_ENABLED(CLOCK_GPIO))
```

Comment:
Formatting needs to match https://docs.mbed.com/docs/mbed-os-handbook/en/5.1/cont/code_style/